fetch.bs

```diff
@@ -6880,7 +6882,15 @@ object), initially null.
 
    <dt>{{RequestInit/window}}
    <dd>Can only be null. Used to disassociate <var>request</var> from any {{Window}}.
-  </dl>
+
+   <dt>{{RequestInit/duplex}}
+   <dd>"half" for initiating a half-duplex (i.e., the user agent send all the request content
+   before starting receiving the response) fetch, and "full" for initiating a full-duplex (i.e.,
+   the user agent starts receiving the response before sending all the request content) fetch.
+   This needs to be set when {{RequestInit/body}} is a {{ReadableStream}}. <span class=note>The
+   semantics for "full" have not been specified yet. See
+   <a href="https://github.com/whatwg/fetch/issues/1254">issue #1254</a>.</span>
```

Comment:
Use `"<code>half</code>"` for formatting. As we don't specify "full" at this point this text needs to be changed a little bit.

Reply:
Done.